addons/web/static/tests/views/fields/progress_bar_field_tests.js

```diff
@@ -334,7 +334,7 @@ QUnit.module("Fields", (hooks) => {
             await nextTick();
             assert.containsNone(
                 target,
-                ".o_progressbar input",
+                "div",
```

Comment:
this diff should not be

Reply:
this is in a separate commit, just to have an error on runbot to demonstrate the effect ;)

Reply:
ho ok sorry :D